tests/snapshots/tests.contrib.asyncpg.test_asyncpg.test_cursor.json

```diff
@@ -0,0 +1,80 @@
+[[
+  {
+    "name": "postgres.connect",
+    "service": "postgres",
+    "resource": "postgres.connect",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "sql",
+    "meta": {
+      "db.name": "postgres",
+      "db.user": "postgres",
+      "out.host": "127.0.0.1",
+      "runtime-id": "dd45decbe18b43409012e9c6a956ddcc"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "out.port": 5432,
+      "system.pid": 67009
+    },
+    "duration": 5844000,
+    "start": 1646993092860981000
+  }],
+[
+  {
+    "name": "postgres.query",
+    "service": "postgres",
+    "resource": "\n        CREATE TEMP TABLE test (id serial PRIMARY KEY, name varchar(12) NOT NULL UNIQUE);\n    ",
```

Comment:
this whitespace gets normalized by the trace agent/backend right?

Reply:
Yes, confirmed with a manual test.